ui/raidboss/data/triggers/twinning.js

```diff
@@ -0,0 +1,143 @@
+'use strict';
+
+// The Twinning
+
+[{
+  zoneRegex: /^The Twinning$/,
+  triggers: [
+    {
+      // The handling for these mechanics is similar enough it makes sense to combine the trigger
+      id: 'Twinning Impact + Pounce',
+      regex: / 1B:........:\y{Name}:....:....:(003[2-5]|005A)/,
+      suppressSeconds: 10,
+      alertText: {
+        en: 'Spread--Avoid cages',
+      },
+    },
+    {
+      id: 'Twinning Beastly Roar',
+      regex: / 14:3D64:Alpha Zaghnal starts using Beastly Roar/,
+      condition: function(data) {
+        return data.role == 'healer';
+      },
+      alertText: {
+        en: 'AoE Damage',
+      },
+    },
+    {
+      id: 'Twinning Augurium',
+      regex: / [23:19:19.606] 14:3D65:Alpha Zaghnal starts using Augurium/,
```

Comment:
Typo here.

Can you dodge Augurium as a tank?

Reply:
That's a good point, I can't be sure so for now we'll set it as non-tanks. Thanks for catching that.